### ISSUE TITLE
Running the command errors on fs.access

### ISSUE THREAD

**Issue by DfKimera** [nebev/ssg-js#4]
After running for the first time, I get the following error:

```
➜  ~  npm install -g ssg-js
/usr/local/bin/ssg -> /usr/local/lib/node_modules/ssg-js/bin/ssg
ssg-js@0.1.3 /usr/local/lib/node_modules/ssg-js
└── blessed@0.1.81
➜  ~  ssg

/usr/local/lib/node_modules/ssg-js/lib/configparse.js:19
    fs.access(filename, fs.R_OK | fs.W_OK, function(err){
       ^
TypeError: Object #<Object> has no method 'access'
    at Object.exports.getConfig (/usr/local/lib/node_modules/ssg-js/lib/configparse.js:19:5)
    at module.exports (/usr/local/lib/node_modules/ssg-js/lib/configdata.js:16:40)
    at Object.exports.run (/usr/local/lib/node_modules/ssg-js/lib/index.js:12:38)
    at Object.<anonymous> (/usr/local/lib/node_modules/ssg-js/bin/ssg:4:6)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
➜  ~
```

**Comment by DfKimera** [comment]
Turns out this is a NodeJS version issue. The `package.json` should be updated to require the minimum version to be newer then 0.12 (via `{ "engines" : { "node" : ">=0.12" } }`)

**Comment by nebev** [comment]
Ah. Awesome mate. Will add that. Thank you.

**Comment by nebev** [comment]
Fixed in 7e35754aa0d506afecdbdc77f7a701224966c0fd
